nifc_wildfires/cli.py

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+import click
+from nifc_wildfires import (
+    get_active_perimeters,
+    get_incidents
+)
+
+
+@click.group()
+def cmd():
+    """
+    A command-line interface for downloading wildfire perimeter and incident points data from NIFC.
+    Returns GeoJSON.
+    """
+    pass
+
+
+@cmd.command(help="Perimeters of active fires in new 2021 feed")
```

Comment:
Let's clip "new" out of here too and make this language a little more evergreen.